protocol/flows/src/v5/txrelay/flow.rs

```diff
@@ -201,6 +201,7 @@ impl RelayTransactionsFlow {
         requests: Vec<RequestScope<TransactionId>>,
         should_throttle: bool,
     ) -> Result<(), ProtocolError> {
+        let last_transactions_transfer = Instant::now();
```

Comment:
Is it intentional that this object is instantiated at the very top, then there's a bunch of awaits in the middle, and then only setting the value at the end?

As opposed to creating `Instant::now()` right before it is used in `self.router.set_last_tx_transfer`?

Reply:
I wanted to avoid as much "noise" (by measuring processing), as possible, ideally I would like to get the ts as soon as we get the message, but as this would be a deeper incursion on the codebase, i decided to put it in as soon as we are in the adequate function.